src/board.jl

```diff
@@ -80,6 +80,84 @@ export attacksto,
     @startboard
 
 
+
+"""
+    MoveList
```

Comment:
I moved this to resolve an error where we used `MoveList` before it was defined (vertically, in this file). I believe I can just do something like 
```julia
MoveList
```
somewhere at the top to register the name in advance. I will try to make this change, if that is what you would like (rather than moving this entire definition around).